### PR TITLE
Move main OBO release file to base variant

### PR DESCRIPTION
**Pull request by matentzn**
This PR fixes https://github.com/monarch-initiative/mondo/issues/7095

1. Changes the main OBO-Format release artefact to the BASE variant of releases.
2. See https://github.com/monarch-initiative/mondo/issues/7095 for context and discussion

### CODE REVIEW

**Comment by matentzn** [comment]
Reviewing this should now be prioritised as this is also fixing some other bugs that I didnt notice before.

**Comment by matentzn** [comment]
@twhetzel can you add this to the April milestone - I cant seem to be able to find out how (maybe there is no milestone yet)

**Comment by twhetzel** [comment]
@matentzn I added it to the April release project board.

File: src/ontology/Makefile [hunk 1]
```diff
@@ -244,10 +244,12 @@ $(ONT).owl: reasoned.owl
...
-$(ONT).obo: reasoned.owl
-	$(ROBOT) annotate -i $< -V $(ONTURI)/releases/`date +%Y-%m-%d`/$(ONT).owl -o $@.tmp.owl
-	owltools --use-catalog $@.tmp.owl --remove-imports-declarations --remove-dangling -o -f obo --no-check $@.tmp && $(obo-filter-axiom-header) $@.tmp > $@
-	rm -f $@.tmp.owl
+$(ONT).obo: $(ONT)-base.owl
+	$(ROBOT) annotate -i $< \
+			-V $(ONTURI)/releases/`date +%Y-%m-%d`/$(ONT).owl \
+			--ontology-iri $(OBO)/$(ONT).owl \
+		convert --check false -f obo -o $@.tmp.obo &&\
+		grep -v ^owl-axioms $@.tmp.obo > $@ && rm $@.tmp.obo
```
Comment (twhetzel) [inline src/ontology/Makefile:252]: looks good

File: src/ontology/Makefile [hunk 2]
```diff
@@ -268,11 +270,15 @@ $(ONT)-with-equivalents.json: $(ONT)-with-equivalents.owl
 # we remove imports and merge in additional assertions
 # note: typically component is not pre-reasoned, but this is compex for mondo.
 
-OTHER_SRC=imports/axioms.owl components/mondo-subsets.owl #components/mondo-tags.owl
+OTHER_SRC=imports/axioms.owl #components/mondo-tags.owl
 
+# The --select "<http://purl.obolibrary.org/obo/BFO_*>" is due to a bug with the reason -x option
+# Which is supposed to exclude external namespaces from reason, but does not. My guess:
+# BFO namespace was needed for Object Properties.
```
Comment (twhetzel) [inline src/ontology/Makefile:277]: thanks for adding this explanation!

**Review by twhetzel — APPROVED** [review]
lgtm

**Comment by matentzn** [comment]
@sabrinatoro we should add to the release notes that the mondo.obo file now contains relationships to genes, antomical and other biological entities.

**Pull request merged**